Code/Tools/FBuild/FBuildCore/WorkerPool/WorkerBrokerage.cpp

```diff
@@ -83,6 +84,7 @@ void WorkerBrokerage::FindWorkers( Array< AString > & workerList )
 
     if ( m_BrokerageRoot.IsEmpty() )
     {
+        FLOG_WARN( "No brokerage root; did you set FASTBUILD_BROKERAGE_PATH?" );
```

Comment:
A pretty common use-case is to have project generation setup to always pass -dist on the command line, but to selectively define the FASTBUILD_BROKERAGE_PATH only for some users.  A great example of this is for a company with  remote workers - you might define the path for local users, but not for remote workers.  This change would cause those users to always get a warning.